index.js

```diff
@@ -90,14 +91,29 @@ async function main(opts) {
     }
   }
 
-  const base = process.env.BASE_URL || 'https://micro.readme.com';
+  // Get the base and remove the trailing slash that comes by default from vercel
+  const base = (process.env.BASE_URL || 'https://micro.readme.com').replace(/\/$/, '');
 
   return axios
     .post(`${base}/api/uploadSpec`, out, {
       headers: { 'X-API-KEY': options.key },
     })
-    .then(() => {
-      log('Successfully synced file to ReadMe Micro! 🦉');
+    .then(response => {
+      const body = response.data;
+      console.log('body', body);
+      if (body.lint?.length) {
+        body.lint.forEach(l => {
+          console.log(`Linting issues in ${l.fileName}:`);
+
+          l.lint.forEach(l => {
+            console.error(` ⚠️ ${l.message} (${l.path.join(' > ')})`);
+          });
+
+          console.log('');
+        });
+        return core.setFailed('Your OAS files failed linting!');
```

Comment:
I haven't checked this (but will), but does `core.setFailed` work w/ Bitbucket pipelines?

Reply:
Yeah that's a great question, i'm actually not sure. I posted this here too: https://github.com/readmeio/readme-micro/blob/7208a5648fb3223d56b8ed70c9dd1f029e02a2f7/index.js#L120-L121

I'm assuming it fails gracefully because this appears to have been working (either that or we just havent hit the error state yet)

Reply:
It works! See the image in the PR description.